app/features/hit_fox_feed.rb

```diff
@@ -0,0 +1,33 @@
+require 'github_feed'
+require 'yajl'
+
+class HitFoxFeed
+  def initialize(user, org, token)
+    @feed = GithubFeed.new(user,org, token)
+    @redis = $redis
+  end
+
+  def update
+    @redis.set 'HitFoxFeed', Yajl::Encoder.new.encode(retrieved_events)
```

Comment:
redis does it automatically, but i remember it was slower than the purposed library.
I can removed it